src/commands/install-deps.yml

```
@@ -45,6 +50,6 @@ steps:
       condition: <<parameters.with-cache>>
       steps:
         - save_cache:
-            key: << parameters.key >>-{{ checksum "<<parameters.app-dir>>/Gemfile.lock"  }}-{{ .Branch }}
+            key: << parameters.key >>-{{ checksum "<<parameters.app-dir>>/Gemfile.lock"  }}-<<#parameters.include-branch-in-cache-key>>{{ .Branch }}-<</parameters.include-branch-in-cache-key>>
```

Should we move the dash inside of the `include-branch-in-cache-key` check? Doing this would mean we could avoid doing anything on line 39 and it should gracefully figure out where the cache is coming from branch or no branch.

It's subtle, but it could look something like:

```suggestion
            key: << parameters.key >>-{{ checksum "<<parameters.app-dir>>/Gemfile.lock"  }}<<#parameters.include-branch-in-cache-key>>-{{ .Branch }}-<</parameters.include-branch-in-cache-key>>
```